docs/content/en/docs/Advanced/build.md

```diff
@@ -279,3 +279,183 @@ gcloud compute images create <new_image_name> --source-uri=<your_bucket>/<cos-gc
 ```
 
 See [here how to use a cloud-init with Google cloud](https://cloud.google.com/container-optimized-os/docs/how-to/create-configure-instance#using_cloud-init_with_the_cloud_config_format).
+
+
+## Build Kairos appliances from scratch
+
+{{% alert title="Note" %}}
+
+This section is reserved for experienced users and advanced use-cases, for instance when building new flavors for Kairos core images. 
+Core Kairos images are pre-configured, optimized images built following the approach described below, which are pre-built and maintained by the Kairos team.
+{{% /alert %}}
+
+Kairos is a meta-distribution as, components of Kairos can be used to build, Immutable, container-based derivatives.
+
+Kairos packages can be used from standard Dockerfiles to build full-fledged Kairos-based derivatives entirely from container images. 
+
+Consider this example which builds an alpine based derivative:
+
+```Dockerfile
```

Comment:
This Dockerfile needs to create a user with a password to make the testing of the image easier (the next step).